main.go

```diff
@@ -212,16 +211,26 @@ func main() {
 	log.Fatal(http.ListenAndServe(*bindAddr, nil))
 }
 
-func parseIssueTemplate(cfg *config) *template.Template {
-	issueTemplate := cfg.IssueBodyTemplate
-	if issueTemplate == "" {
-		issueTemplate = DefaultIssueBodyTemplate
+// parseTemplate parses a template file, with fallback to default.
+//
+// If `configFileSettingValue` is non-empty, it is used as the name of a file to read. Otherwise, `defaultTemplate` is
+// used.
+//
+// The template text is then parsed into a template named `templateName`.
+func parseTemplate(defaultTemplate string, configFileSettingValue string, templateName string) *template.Template {
```

Comment:
`configFileSettingValue` feels like a bit of a mouthful. Do we really care that the template file path comes from the config? I'd rename this to `templateFilePath` personally.